apps/core-frontend/src/ui/fixedrates/FixedRateCardList/FixedRateCardList.tsx

```diff
@@ -1,14 +1,24 @@
 import { PrincipalTokenInfo } from "@elementfi/core-tokenlist";
-import tw from "efi-tailwindcss-classnames";
+import { aztecTerms } from "integrations/aztec/aztecAddresses";
 import { ReactElement } from "react";
+import { jt } from "ttag";
 import { useIsTailwindLargeScreen } from "ui/base/mediaBreakpoints";
 import { FixedRateCardWithZap } from "ui/fixedrates/FixedRateCard/FixedRateCardWithZap";
+
+import tw from "efi-tailwindcss-classnames";
+
 import { FixedRateCardListHeader } from "./FixedRateCardListHeader";
 
 interface FixedRateCardListProps {
   principalTokens: PrincipalTokenInfo[];
 }
 
+const aztecLink = (
+  <a href="https://zk.money/earn" key="aztec-link">
+    zk.money
+  </a>
```

Comment:
open in new tab?

Reply:
Not needed here. Navigating to zk.money doesn't interrupt any processes on the current page, eg: user isn't half-finished filling out a form or waiting for a download, etc..

 Default web link behavior here is fine :)